defaults/serverConfig/server.json

```diff
@@ -22,13 +22,7 @@
         "override": true,
         "value": "sameorigin"
       }
-    },
-    "childProcesses": [
-      {
-        "path": "../bin/zssServer.sh",
```

Comment:
can you restore this, since it is harmless in the dev environment? we realized that we needed to remove it in zowe-install-packaging, but here it is fine.